challenges/test/lib/src/test/java/code/BinarySearchTest.java

```diff
@@ -0,0 +1,41 @@
+package code;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class BinarySearchTest {
+  @Test public void testBinarySearchFile(){
+    BinarySearch testMethod = new BinarySearch();
+    assertTrue(testMethod.testBinarySearch());
+  }
+
```

Comment:
Great work running multiple tests, don't forget to test for the base cases however! What if the array was empty or target value wasn't found!